### DISCUSSION TITLE
ADC_VCC not work on my ESP8266

### DISCUSSION THREAD

**Discussion by tibinoo** [arendst/Tasmota#21759] [Q&A] [answered]
Hi
I compiled my own image with #define USE_ADC_VCC 
But in MQTT or console log I don's see any Vcc item.
Why it not work, or how can i test it?
Is any command to view Vcc item?
I tried it on four esp8266 board and not works anywhere, but when I used before lua nodemcu firmware then VCC shoed good.

Version:
Program Version | 14.1.0.3 (sensors)
Build Date & Time | 2024-07-09T10:05:49
Core/SDK Version | 2_7_7/2.2.2-dev(38a443e)
ESP Chip Id | 1280330 (ESP8266EX)

From LOG Console:
10:46:36.663 MQT: tele/Cidlo12/STATE = {"Time":"2024-07-10T10:46:36","Uptime":"0T14:00:12","UptimeSec":50412,"Heap":19,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":31,"MqttCount":3,"Wifi":}

**Comment by sfromis** [comment, marked as answer]
Looks like you tried adding a feature to one of the precomposed build variant (sensors), which is not a supported thing to do, as mentioned here: https://tasmota.github.io/docs/Compile-your-build/#customize-your-build

When succeeding to add the feature (instead of analog), you should indeed see the value in the `STATE` payloads, like `"Vcc":3.475`

**Comment by tibinoo** [comment]
When I compiled standart build (without sensors) it works fine 
Thanks
13:11:18.258 MQT: tele/Cidlo12/STATE = {"Time":"2024-07-10T13:11:18","Uptime":"0T00:00:13","UptimeSec":13,"Vcc":3.466,"Heap":27,"SleepMode":"Dynamic
